fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/data/LoanProductData.java

```diff
@@ -117,6 +117,7 @@
     private final Integer minimumDaysBetweenDisbursalAndFirstRepayment;
     private final boolean canDefineInstallmentAmount;
     private final Integer installmentAmountInMultiplesOf;
+	private final boolean canAutoAllocateOverpayments;
```

Comment:
I believe, when canAutoAllocateOverpayments is enabled from loan product definition, then we need to mandate linked savings account for the loans created from this product